knowage-core/src/main/java/it/eng/spagobi/api/AbstractDocumentResource.java

```diff
@@ -167,7 +167,7 @@ protected Object getObjectIdentifier(String labelOrId) {
 		return documentIdentifier;
 	}
 
-	public Response getDocumentTemplate(String label) {
+	protected Response getDocumentTemplate(String label) {
```

Comment:
If possible, I would prefer private but this is an abstract class, we need a check.

Reply:
This one can't be private, it is being used in DocumentResource.java (two files, one of them is v2).